canonical-web-and-design/hiring.canonical.com
# Display hiring lead dropdown only in debug mode

## Done
Hiring lead dropdown is only displayed when application is in development mode.

## QA
Setup:
- Add "DEVEL=true" to the .env.local 
- Run `dotrun`.
- Verify hiring lead dropdown is displayed.
- Stop the application
-  Update "DEVEL=false" to the .env.local 
- Run `dotrun`.
- Verify hiring lead dropdown is hidden.

## Issue
Fixes #127
